### ISSUE TITLE
Currency doesn't show up on the character sheet unless I click "Edit"

### ISSUE THREAD

**Issue by KlumsyKyller** [larrychristensen/orcpub#114]
Exactly as the title says. To find my currency, I have to edit my character and go to equipment. It shows up in the "Treasure" section. When I am not editing, however, there is no "Treasure" section, and gold doesn't show up anywhere else.

**Comment by XercinVex** [comment]
Yes this should all be available on the view page and regular character list but it is very hard to find because you have to go into edit or print views
